### ISSUE TITLE
import command  for azurerm_api_management_api_policy is wrong

### ISSUE THREAD

**Issue by juntingxiao** [hashicorp/terraform-provider-azurerm#24034]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment and review the [contribution guide](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/README.md) to help.

### Terraform Version

1.0.0

### AzureRM Provider Version

3.0.0

### Affected Resource(s)/Data Source(s)

azurerm_api_management_api_policy 

### Terraform Configuration Files

```hcl
data "azurerm_api_management_api" "example" {
  name                = "my-api"
  api_management_name = "example-apim"
  resource_group_name = "search-service"
  revision            = "2"
}

resource "azurerm_api_management_api_policy" "example" {
  api_name            = data.azurerm_api_management_api.example.name
  api_management_name = data.azurerm_api_management_api.example.api_management_name
  resource_group_name = data.azurerm_api_management_api.example.resource_group_name

  xml_content = <<XML
<policies>
  <inbound>
    <find-and-replace from="xyz" to="abc" />
  </inbound>
</policies>
XML
}
```

### Debug Output/Panic Output

```shell
data "azurerm_api_management_api" "example" {
  name                = "my-api"
  api_management_name = "example-apim"
  resource_group_name = "search-service"
  revision            = "2"
}

resource "azurerm_api_management_api_policy" "example" {
  api_name            = data.azurerm_api_management_api.example.name
  api_management_name = data.azurerm_api_management_api.example.api_management_name
  resource_group_name = data.azurerm_api_management_api.example.resource_group_name

  xml_content = <<XML
<policies>
  <inbound>
    <find-and-replace from="xyz" to="abc" />
  </inbound>
</policies>
XML
}
```

### Expected Behaviour

terraform import azurerm_api_management_api_policy.example /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/group1/providers/Microsoft.ApiManagement/service/service1/apis/exampleId/policies/policy

### Actual Behaviour

terraform import azurerm_api_management_api_policy.example /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/group1/providers/Microsoft.ApiManagement/service/service1/apis/exampleId

### Steps to Reproduce

terraform import azurerm_api_management_api_policy.example /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/group1/providers/Microsoft.ApiManagement/service/service1/apis/exampleId
╷
│ Error: parsing Resource ID "/subscriptions/2e375de1-7a40-4fe1-9b1a-edca808402a4/resourceGroups/rg-sharesvc-dev-apim-eas-01/providers/Microsoft.ApiManagement/service/apim-sharesvc-dev-apim-eas-01/apis/6524f82d5c9a39c23e477e19": ID was missing the `policies` element
│
│
╵

### References

import command  for azurerm_api_management_api_policy is wrong 
need to amend  /policies/policy after the /apis/exampleId

**Comment by sinbai** [comment]
Hi @juntingxiao thanks for opening this issue. In fact, there is a change about import command for `azurerm_api_management_api_policy` from v3.71.0 of terraform provider. Before v3.71.0, the command included " /policies/policy", and from v3.71.0, the command did not include " /policies/policy". Please see the terraform documentation [v3.68.0](https://registry.terraform.io/providers/hashicorp/azurerm/3.68.0/docs/resources/api_management_api_policy#import) and [3.71.0](https://registry.terraform.io/providers/hashicorp/azurerm/3.71.0/docs/resources/api_management_api_policy#import) for details.

**Comment by rcskosir** [comment]
Thank you for taking the time to raise this! I am going to close this with @sinbai‘s response as an answer. 
If you have future questions, I suggest using the [Community Resources](https://www.terraform.io/community), such as the [Azure Provider forum](https://discuss.hashicorp.com/c/terraform-providers/tf-azure/34).
